server/adapters/groups.js

```diff
@@ -0,0 +1,32 @@
+var Sequelize = null;
```

Comment:
I think it's FAR better to use three require than just set three variable in the file. And then you can get rid of the init() function.